src/main/java/net/datafaker/providers/base/DateAndTime.java

```diff
@@ -264,6 +273,25 @@ public Timestamp birthday(int minAge, int maxAge) {
             LocalDateTime.of(date, LocalTime.ofNanoOfDay(faker.number().numberBetween(0, DAYS_NANOS))));
     }
 
+    /**
+     * Generates a random birthday between two ages from now as LocalDate.
+     *
+     * @param minAge the minimal age
+     * @param maxAge the maximal age
+     * @return a random birthday between {@code minAge} and {@code maxAge} years ago from now.
+     * @throws IllegalArgumentException if the {@code maxAge} is lower than {@code minAge}.
+     */
+    public LocalDate birthdayLocalDate(int minAge, int maxAge) {
+        final LocalDate localDate = LocalDate.now();
+        final LocalDate from = localDate.minusYears(maxAge);
+        if (minAge == maxAge) {
+            return from;
+        }
```

Comment:
Do this before instantiating any other cars?

Reply:
Should we check that neither value is negative?

Reply:
it works ok with negative numbers (will return birthday in future)

Reply:
Maybe note that in the javadoc then?

Reply:
Also fixed my original note here should have said “vars” instead of “cars” 🤦‍♂️

Reply:
added to javadoc